src/mqt/bench/benchmarks/vqerealamp.py

```diff
@@ -24,20 +25,27 @@
     from qiskit.circuit import QuantumCircuit
 
 
-def create_circuit(num_qubits: int) -> QuantumCircuit:
-    """Returns a quantum circuit implementing the RealAmplitudes ansatz with random parameter values.
+def create_circuit(num_qubits: int, reps: int = 3, random_parameters: bool = True) -> QuantumCircuit:
```

Comment:
Is this also the default for the Qiskit function?
I think we should be matching Qiskit's default for all of these algorithms

Reply:
Done.